docs/examples/04_backends_vrep/01_simulation_with_vrep.rst

```diff
@@ -67,9 +67,11 @@ forward kinematics:
         robot_b = rfl.Robot('B')
         client.set_robot_config(robot_a, config_robot_a)
         client.set_robot_config(robot_b, config_robot_b)
+        group_a = robot_a.model.attr['index']
+        group_b = robot_b.model.attr['index']
 
-        frame_a = client.forward_kinematics(robot_a)
-        frame_b = client.forward_kinematics(robot_b)
+        frame_a = client.forward_kinematics(None, None, group=group_a)
```

Comment:
Maybe we can refresh the `rfl` module to have `Rfl()` as a fixed, only-for-examples, constructor of a minimal fake model, so that this line has less `None`s, and it looks something like 
```python
robot = Rfl()
client.forward_kinematics(rfl, None, group=group_a)
```

It would also be possible to add configuration to be used on the FK call.